deploy/kubernetes/examples/demo-volumereplication-ear.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: replication.storage.openshift.io/v1alpha1
+kind: VolumeReplication
+metadata:
+  name: demo-volumereplication
+  namespace: default
+spec:
+  volumeReplicationClass: demo-volumereplicationclass
+  replicationState: primary
```

Comment:
can you add the optional values for replicationState? like in [storagecalss ](https://github.com/IBM/ibm-block-csi-driver/blob/226fcc88e5642588ed44d590b53f350e07e3e50c/deploy/kubernetes/examples/demo-storageclass.yaml#L14)

Reply:
possible values added